cmd/kyverno/main.go

```diff
@@ -38,20 +38,20 @@ import (
 const resyncPeriod = 15 * time.Minute
 
 var (
+	//TODO: this has been added to backward support command line arguments
+	// will be removed in future and the configuration will be set only via configmaps
```

Comment:
is this actually true, or will we always support both flags and CM for settings?

Reply:
We've been planned on it a while back, track in https://github.com/kyverno/kyverno/issues/261.